### PR TITLE
Merge Getting Started code snippet into master

### PR DESCRIPTION
**Pull request by dersim-davaod**
We need to merge some changes from `dev` into master to make them available for `ios-sdk` repository, as it has `mapbox-vision-ios` as a submodule.
The changes were introduced in scope of https://github.com/mapbox/mapbox-vision-ios/pull/283/files

Checks:

* [ ] Update changelog
* [x] Rebase to `dev` branch
* [x] Assign reviewers

Linked issues: 
#283 
https://github.com/mapbox/ios-sdk/pull/899

### CODE REVIEW

File: Examples/VisionExamples/Code Snippets/GettingStartedViewController.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // create a video source obtaining buffers from camera module
+        videoSource = CameraVideoSource()
+        
+        // create VisionManager with video source
+        visionManager = VisionManager.create(videoSource: videoSource)
+        // set up the `VisionSafetyManagerDelegate`
```
Comment (chezzdev) [inline Examples/VisionExamples/Code Snippets/GettingStartedViewController.swift]:
```suggestion
        // set up the `VisionManagerDelegate`
```

File: Examples/VisionExamples/Code Snippets/GettingStartedViewController.swift [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        visionManager.delegate = self
+        
+        // create VisionARManager to use AR features
+        visionARManager = VisionARManager.create(visionManager: visionManager)
+        // set up the `VisionSafetyManagerDelegate`
```
Comment (chezzdev) [inline Examples/VisionExamples/Code Snippets/GettingStartedViewController.swift]:
```suggestion
        // set up the `VisionARManagerDelegate`
```

File: Examples/VisionExamples/Code Snippets/GettingStartedViewController.swift (hunk 1 above)
Comment (dersim-davaod) [reply Examples/VisionExamples/Code Snippets/GettingStartedViewController.swift]: Very nice catch! Fixed.

File: Examples/VisionExamples/Code Snippets/GettingStartedViewController.swift (hunk 2 above)
Comment (dersim-davaod) [reply Examples/VisionExamples/Code Snippets/GettingStartedViewController.swift]: Fixed.

**Comment by dersim-davaod** [comment]
Per agreement with @chezzdev as these changes were reviewed, they're merged now.
The little difference we introduced will be merged into `dev` tomorrow morning.

**Pull request merged**